HOK.Utilities/HOK.ColorBasedIssueFinder/IssueFinder.xaml.cs

```diff
@@ -0,0 +1,408 @@
+﻿using Autodesk.Revit.DB;
```

Comment:
I don't think this is something to address immediately, but I think it'd be good to extract Revit-related logic from this file and keep it primarily for UI-related logic like click handling, color selection, etc.